doc/user_guide/ensembleModel.tex

```diff
@@ -0,0 +1,229 @@
+the name of the model goes in the \emph{text} of the \xmlNode{Model} node, right after the first closing angle
+bracket.  The subnodes of the \xmlString{ballistics} model are the \xmlNode{Input}, which points to the input
+template input file we defined in the \xmlNode{Files} block; and the \xmlNode{TargetEvaluation}, which points
+to the \xmlNode{PointSet} we defined in the \xmlNode{DataObjects} block.  These are critical to allowing the
+Ensemble Model both to run the \xmlString{ballistics} model correctly, as well as the entire ensemble in a
+sensible manner.
+
+In general, some models (such as a \xmlNode{ROM} or \xmlNode{ExternalModel}) do not require any files as
+inputs.  In this case, the \xmlNode{Input} for an input-less model should be a placeholder data object,
+usually with the same input space as the \xmlNode{TargetEvaluation} data object, and a placeholder output.
+For more information, see examples running external models in these guides as well as the user manual.
+
+After the \xmlString{ballistics} model, we see a similar structure pointing to the \xmlString{impact} model.
+The only differences between this model specification and the ballistics model specificatoin are the names of
+the \xmlNode{Input} file and \xmlNode{TargetEvaluation} data object.
+
+That's it!  With the two models defined, and the Ensemble Model assembled, no more work is required to
+prepare these two models to run.  More complicated systems may require additional nodes in the ensemble model;
+see the user manual for details.
+
+\subsubsection{Steps}
+Finally, to put all the pieces together, we consider the \xmlNode{Steps} node:
+\xmlExample{framework/user_guide/EnsembleModel/basic.xml}{Steps}
+We take two steps in this simulation, one \xmlNode{MultiRun} called \xmlString{sample} to sample the ensemble model on a grid,
+and one \xmlNode{IOStep} to print the results.  These are constructed just as they would be with any other
+model; as far as the \xmlNode{Steps} are concerned, the ensemble model is like any other model.  Note that we
+include the two template input files as \xmlNode{Input} nodes for the sampling step.
+
+We do not discuss the \xmlNode{Distributions}, \xmlNode{Samplers}, or \xmlNode{OutStreams}
+here.  These operate in general as they would in any other RAVEN input file.  We will note, however, that the
+\xmlNode{Grid} sampler named \xmlString{grid} samples all the inputs that are needed by either of the submodels,
+unless the inputs are provided by another model.  In this case, that means we need to sample
+$y_0,\theta_0,v_0,m,r$ but we do not sample $E$ since it is calculated by the \xmlString{ballistics} model.
+
+We can find all the sampled and calculated values from both of the models in the output produced by this run,
+found at
+\begin{verbatim}
+raven/tests/framework/user_guide/EnsembleModel/run_basic/results.csv
+\end{verbatim}
+
+
+
+\subsection{External Models in Ensemble Models}
```

Comment:
```ExternalModels in EnsembleModels```

Reply:
See above.  Surely we can do better than using code in sentences.

Reply:
We use this approach throughout the user_manual... It is a consistency request.

Reply:
I see.  I was under the impression we wanted the user guide to be easier to follow.  I will make your changes.

Reply:
I do not see how this modification can make the user guide difficult to follow...